tests/aes_encryption.rs

```diff
@@ -0,0 +1,78 @@
+use std::io::{self, Read};
+use zip::ZipArchive;
+
+const SECRET_CONTENT: &str = "Lorem ipsum dolor sit amet";
+
+const PASSWORD: &[u8] = b"helloworld";
```

Comment:
Are there restrictions on the characters possible in the password?
I am thinking upper cased letters, punctuation, digits and so on.

Reply:
`aA1!@#$%^&*()_+=-\|}]{[` works as a password. In general anything should be possible, since passwords are just a slice of bytes. I wanted to try a password with an emoji, but the hard part is encryption, since p7zip only supports ascii.